docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -456,6 +459,16 @@ An example:
 
 This will trigger all the nodes that connects to the _GuestAccess_ SSID to use inline enforcement mode (PacketFence will return a void VLAN or the `inlineVlan` if defined in switch configuration) and the MAC address `00:11:22:33:44:55` client if it connects on another SSID.
 
+Technical introduction to RADIUS enforcement
+--------------------------------------------
+
+Introduction
+~~~~~~~~~~~~
+
+The concept of having a RADIUS enforcement is to not use registration and isolation, nor the portal. Everything here is for 802.1X authentication only. By default your management interface will be your RADIUS interface, if needed you can add another one via 'Network Configuration->Networks->Interfaces' adding an `Other` type of interface and the additionnal daemon `RADIUS`.
```

Comment:
The concept of having a RADIUS enforcement is to not use registration and isolation, nor the portal. Everything here is for RADIUS authentication only. By default the management interface will be the RADIUS interface. If needed, it is possible to add another one via 'Network Configuration->Networks->Interfaces' adding an `Other` type of interface and the additional daemon `RADIUS`.